emails/2022-09-30.md

```diff
@@ -0,0 +1,34 @@
+
+Well known fact, I'm from Canada. I've lived in the US for almost 20 years and I became an American last year. Whenever Dolt is acting up, you can definitely ["Blame Canada"](https://www.youtube.com/watch?v=bOR38552MJA).
+
+### New Format is the Default
+
+I know. I know. You're tired of hearing about this. Problem is, we have to get everyone migrated to the new format. It's just better. 
+
+It's [faster](https://docs.dolthub.com/sql-reference/benchmarks/latency) and [has fewer bugs](https://github.com/dolthub/dolt/issues/4419). We're starting to get people asking [why it's not the default already](https://github.com/dolthub/dolt/issues/4419#issuecomment-1261527471)! Sorry Kyra.
+
+The new format is now the default [on DoltHub](https://www.dolthub.com/blog/2022-09-19-new-format-dolthub/) and [in Dolt]() for new databases. You can migrate an old format database using `dolt migrate`. A migrate button on DoltHub is coming soon, hopefully by Friday next week. Like Canada Geese, start migrating to the warmer new format for the winter.
+
+[![Migrate](../images/canada-geese.png)]()
+
+### Hosted Just Keeps Getting Better
+
+This week we launched [Pull Requests on Hosted](https://www.dolthub.com/blog/2022-09-26-hosted-ui-writes/). If you haven't tried Hosted Dolt, definitely check it out and let us know what you think. If you're price sensitive, just reply to this email and I'll mark your instance free trial.
+
+I think it's important to explain how we think of Dolt, DoltHub/DoltLab, and Hosted Dolt. When we started building Dolt and DoltHub, we had the Git and GitHub model firmly entrenched in our mind. Dolt was [Git for Data](https://www.dolthub.com/blog/2020-03-06-so-you-want-git-for-data/). DoltHub was Github for Dolt. DoltHub would be offline and be a collaboration/sharing hub for data. DoltHub and DoltLab are still that and they work if that is what you need.
+
+But as Dolt evolved into a [version controlled database](https://www.dolthub.com/blog/2022-08-04-database-versioning/), we searched for database metaphors to describe DoltHub. The best way to think about DoltHub in the database context is a user interface to manage snapshots (ie. backups) and, oh, snapshots are editable and merge-able. This was useful but we don't think it's what people want for a version controlled database. 
+
+People want a production database that they can clone, branch, merge, and diff. They want to make pull requests against the production database. The production database is the main branch AND the collaboration hub. Bug in production? Clone production to your laptop to debug. Schema change? Make a branch of production and test it. Pull request with peer review to get it merged. That's what [Hosted Dolt](https://hosted.doltdb.com/) is. It's a production database with Git-style versioning. It changes the game. It's just a better Hosted MySQL.
+
+[![Which Dolt is right for you?](../images/dolt-options.png)](https://hosted.doltdb.com/)
```

Comment:
Want me to quickly make a link to that chart on the homepage? Can be `https://hosted.doltdb.com#which-dolt-is-right-for-you`?

Reply:
That would be great.

Reply:
Kk will be done before I leave today